Userland/Libraries/LibGUI/RegularEditingEngine.cpp

```diff
@@ -57,10 +57,10 @@ void RegularEditingEngine::sort_selected_lines()
 
     auto& lines = m_editor->document().lines();
 
-    auto start = lines.begin() + (int)first_line;
-    auto end = lines.begin() + (int)last_line + 1;
+    auto start = (int)first_line;
```

Comment:
I know this is preexisting but what's going on with these casts? Why are we declaring these variables as size_t just to turn around and cast them to int? That `get_line_selection_boundaries` function with two out params and no return value looks quite non-idiomatic.

Reply:
I can fix up the cast but everything else is a yak for another day